source/connect/authentication-connection.txt

```diff
@@ -13,3 +13,184 @@ Authentication Connection Tab
+
+      .. note:: 
+
+         X.509 Authentication requires a client certificate. To enable 
+         TLS and add a client certificate, see the :ref:`TLS / SSL tab <tls-ssl-tab>`
+         in |compass-short|.
+
+      .. _kerberos-authentication:
+
+      Kerberos
+      ~~~~~~~~
+
+      Select :guilabel:`Kerberos` if the deployment uses :manual:`Kerberos 
+      </core/kerberos/>` as its authentication mechanism. 
+
+      Provide the following information:
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 50, 50
+
+         * - Field
+           - Description
+
+         * - Principal
+           - Every participant in the authenticated communication is known as a 
+             "principal", and every principal must have a unique name. 
+
+         * - (Optional) Service Name
+           - Every MongoDB :binary:`mongod` and :binary:`mongos` instance 
+             (or exe or exe on Windows) must have an associated service name. The 
+             default is ``mongodb``. 
+
+         * - (Optional) Canonicalize Host Name
+           - Kerberos uses the canonicalized form of the host name ``(cname)`` 
+             when constructing the principal for |compass|. 
+
+         * - (Optional) Service Realm 
+           - The service realm is the domain over which a Kerberos authentication 
+             server has the authority to authenticate a user.
+
+             If you choose to :guilabel:`Canonicalize Host Name`, you have the 
+             following options:
+
+             .. list-table::
+                :header-rows: 1
+                :widths: 50 50
+
+                * - Option
+                  - Description 
+
+                * - Forward
+                  - The driver does a ``cname`` lookup on the kerberos hostname.
+
+                * - Forward and Reverse
+                  - Performs a forward DNS lookup and then a reverse lookup on that 
+                    value to canonicalize the hostname.
+                    
+             For more information on principal name canonicalization in Kerberos, see 
+             this `RFC document <https://tools.ietf.org/html/rfc6806.html>`__.
+      
+         * - (Optional) Provide password directly
+           - Used to verify your identity. 
+
+      .. _ldap-authentication:
+
+      LDAP
+      ~~~~
+
+      Select :guilabel:`LDAP` if the deployment uses :manual:`LDAP 
+      </core/security-ldap-external/>` as its authentication mechanism. 
+
+      Provide the following information:
+
+      - Username
+
+      - Password
+
+      .. _aws-iam-authentication:
+
+      AWS IAM  
+      ~~~~~~~
+
+      Select :guilabel:`AWS IAM` if the deployment uses `AWS IAM 
+      <https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html>`__ as 
+      its authentication mechanism. 
+      
+      The following fields are optional as they are defined on your platform 
+      using their respective AWS IAM environment variables. :mongosh:`mongosh <>`
```

Comment:
It's the node driver that Compass uses which does the environment variable checking here. Maybe better to just put Compass though.
```suggestion
      using their respective AWS IAM environment variables. |compass|
```

Reply:
Done.